### DISCUSSION TITLE
Ability to hide user menu and total control of the items

### DISCUSSION THREAD

**Discussion by bmspereira-07** [filamentphp/filament#13909] [Ideas]
Hey everyone! 

As @danharrin suggested, I'm creating this discussion for a new feature that involves the user menu dropdown.

My ideas are:

1- Hide completely the user menu. For example the client doesn't want the dropdown menu, just an icon button to logout and before that just the name of the user or avatar.

2- Hide the default items (profile, theme switcher, logout) and the user manages the items via userMenuItems([]).

Have a great day.

**Comment by danharrin** [comment]
Once v4 is released, I would accept a PR with these changes

**Reply by bmspereira-07** [reply]
Looking forward to it!
